docs/hugo/content/contributing/create-a-new-release.md

```diff
@@ -17,21 +22,88 @@ title: Creating a new release
 There may be breaking changes in a new release of ASO, either due to changes we made or changes made in the upstream
 service Swagger specifications. We must validate each breaking change so that we can notify customers about it.
 
-1. Download the latest Helm chart release (example [beta.3](https://github.com/Azure/azure-service-operator/blob/main/v2/charts/azure-service-operator-crds-v2.0.0-beta.3.tgz)) and extract it.
-2. Run `task controller:gen-helm-chart` to generate a local helm chart (saved to `v2/charts`). Extract it as well.
-3. Produce a diff between these files: `diff -u <old> <new> > comparison.diff` and examine it.
+1. Download the CRD definitions for the prior release (in this example, v2.2.0):
+
+   ``` bash
+   wget https://github.com/Azure/azure-service-operator/releases/download/v2.2.0/azureserviceoperator_v2.2.0.yaml
+   ```
+
+2. Download the CRD definitions for the new release (in this example, v2.3.0):
+
+   ``` bash
+   wget https://github.com/Azure/azure-service-operator/releases/download/v2.3.0/azureserviceoperator_v2.3.0.yaml
+   ```
+
+   This will only be available when the GitHub action trigged by publishing the release has finished.
```

Comment:
We should do the diff against a local copy rather than the published copy IMO, since the intent is to catch diffs we didn't expect, and/or help crafting the release notes, both of which you have to do before the release itself.

I think you can do `task controller:run-kustomize` and then look at the `bin/azureserviceoperator_customresourcedefinitions_{{.VERSION}}.yaml` file.

That file will _only_ have the CRDs, so it compares with the `azureserviceoperator_customresourcedefinitions_v2.2.0.yaml` included in each release.